README.md

```diff
@@ -609,6 +609,25 @@ enabled = "false"
 port = "9930"
 ```
 
+### Recommended settings for development
+
+The default configuration settings are biased toward running in production environments.
+Although Proxy attempts to detect the environment and set a sensible default for logging, your mileage may vary.
+
+To turn on human-friendly logging:
+
+```bash
+CS_LOG__FORMAT = "pretty"
+CS_LOG__ANSI_ENABLED = "true"
+```
+
+If you are frequently changing the database schema or making updates to the column encryption configuration,
+it can be useful to reload the config and schema more frequently.
```

Comment:
```suggestion
If you are frequently changing the database schema or making updates to the column encryption configuration, it can be useful to reload the config and schema more frequently:
```